app/main.py

```diff
@@ -2,0 +1,19 @@
+from functools import reduce
+from app.players.player import Player
+from app.players.elves.elf import Elf
+from app.players.dwarves.dwarf import Dwarf
+
+
+def calculate_team_total_rating(players: list[Player]) -> int:
+    return reduce(lambda accum,
+        player: accum + player.get_rating(), players, 0)
+
+
+def elves_concert(elves: list[Elf]) -> None:
+    for elf in elves:
+        elf.play_elf_song()
+
+
+def feast_of_the_dwarves(dwarves: list[Dwarf]) -> None:
+    for dwarf in dwarves:
+        dwarf.eat_favourite_dish()
```

Comment:
The `feast_of_the_dwarves` function correctly iterates over the list of dwarves and calls the `eat_favourite_dish` method. Ensure that all dwarf classes have this method implemented.